upload/views/empty_upload.py

```diff
@@ -91,7 +104,18 @@ def post(self, request, *args, **kwargs):
             if any(map(lambda regex: regex.match(file), compiled_files_to_ignore))
         ]
 
-        if set(changed_files) == set(ignored_changed_files):
+        if should_force is True:
```

Comment:
Maybe move this before you do the `changed_files` scan? No point in doing that if you're going to force the notif regardless